src/App.js

```diff
@@ -0,0 +1,38 @@
+import { BrowserRouter as Router, Routes, Route } from 'react-router-dom';
+import './App.css';
+import { useDispatch } from 'react-redux';
+import { useEffect } from 'react';
+import Navigation from './components/Navbar/navigation';
+import Profile from './components/profile/Profile';
+import Missions from './components/missions/missions';
+import Rockets from './components/rockets/Rocket';
+import { fetchDragons } from './redux/Dragons/dragonSlice';
+import { fetchMissions } from './redux/Missions/missionsSlice';
+import { fetchRockets } from './redux/Rockets/rocketSlice';
+import Dragon from './components/dragons/Dragon';
+
+function App() {
+  const dispatch = useDispatch();
+  useEffect(() => {
+    dispatch(fetchMissions());
+    dispatch(fetchDragons());
+    dispatch(fetchRockets());
+  }, [dispatch]);
```

Comment:
- [ ] Hi Team, As mentioned in the requirements :point_down: 
![image](https://user-images.githubusercontent.com/97521557/225405524-b2f68e36-413d-49ae-9170-2a9866fc2f1e.png)

Please make sure that you dispatch the actions only when needed, so fetch rockets when you initially render the page then fetch dragons on dragons page and missions on missions page, remove this useEffect and dispatch from here and only use it inside your components so whenever they render the useEffect will fetch the data, you should add a conditional check also so it does not fetches data on every re-render, you can[ use redux dev tools](https://chrome.google.com/webstore/detail/redux-devtools/lmhkpmbekcpmknklioeibfkpmmfibljd?hl=en) to keep track of the actions you are dispatching